packages/tailwindcss/src/intellisense.ts

```diff
@@ -33,7 +34,7 @@ export function getClassList(design: DesignSystem): ClassEntry[] {
     }
   }
 
-  list.sort((a, b) => (a[0] === b[0] ? 0 : a[0] < b[0] ? -1 : 1))
+  list.sort(([a], [b]) => compare(a, b))
```

Comment:
I added this, but I wonder if it's better todo this instead:
```suggestion
  list.sort((a, b) => compare(a[0], b[0]))
```
🤔